src/Servers/IIS/AspNetCoreModuleV2/CommonLib/HostFxrResolver.cpp

```diff
@@ -93,7 +94,12 @@ HostFxrResolver::GetHostFxrParameters(
             LOG_INFOF(L"Checking application.dll at '%ls'", applicationDllPath.c_str());
             if (!is_regular_file(applicationDllPath))
             {
-                throw InvalidOperationException(format(L"Application .dll was not found at %s", applicationDllPath.c_str()));
+                errorContext.subStatusCode = 38;
+                errorContext.errorReason = "Application dll not found. Confirm the application dll is present. Single-file deployments are not supported in IIS.";
+                errorContext.generalErrorType = "ANCM Application Dll Not Found";
+                throw InvalidOperationException(
+                    format(L"Application .dll was not found at %s. Confirm the application dll is present. Single-file deployments are not supported in IIS.",
```

Comment:
Do we not want the "at %s" in the error reason?

Guess that could be exposing info to clients?

Reply:
These should only be showing in development anyway though right?

Reply:
The exception isn't logged to the response at all. Only Event logs.

Reply:
Not talking about the exception

Reply:
Sorry, misread. Yes, that would be exposing data to clients. I'll add it in detailed information though.